pkg/kubevirt/client.go

```diff
@@ -27,10 +28,22 @@ import (
 )
 
 const (
-	vmiSubresourceURL       = "/apis/subresources.kubevirt.io/%s/namespaces/%s/virtualmachineinstances/%s/%s"
-	annDefaultSnapshotClass = "snapshot.storage.kubernetes.io/is-default-class"
+	vmiSubresourceURL               = "/apis/subresources.kubevirt.io/%s/namespaces/%s/virtualmachineinstances/%s/%s"
+	annDefaultSnapshotClass         = "snapshot.storage.kubernetes.io/is-default-class"
+	InfraStorageClassNameParameter  = "infraStorageClassName"
+	InfraSnapshotClassNameParameter = "infraSnapshotClassName"
 )
 
+type InfraTenantStorageSnapshotMapping struct {
+	VolumeSnapshotClasses []InfraToTenantMapping `yaml:"volumeSnapshotClasses,omitempty"`
+	StorageClasses        []InfraToTenantMapping `yaml:"storageClasses"`
```

Comment:
As this is written now, I don't think you really need the StorageClasses infra to tenant mapping, do you?  We just need to know what Tenant VSC is compatible with what infra SC.